src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.dao;
+
+import java.util.Objects;
+
+public class FruitDao {
+    private String name;
+    private int quantity;
+
+    public FruitDao(String name, int quantity) {
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
```

Comment:
Consider adding validation in the `setQuantity` method to ensure that the quantity is not negative. This will help maintain data integrity.